decidim/decidim
# Upgrade paper_trail from 12.3 to 15.1

#### :tophat: What? Why?
This PR is updating PaperTrail from version 12.3 to 15.1 so that we can have Rails 7.1 support. 
While migrating this, i have noticed that the changeset column is being stored as text, and i have converted it to JSON. 
This is done due to fact that PaperTrail 13.0 introduced a Breaking Change in the YAML interaction which affected from the security point of view. https://github.com/paper-trail-gem/paper_trail/blob/master/doc/pt_13_yaml_safe_load.md

![image](https://github.com/user-attachments/assets/04e9a6a2-9403-431b-892d-f53fbede3033)

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #2206

#### Testing
1. Create a new development application , or seed the db using develop branch
2. apply this patch & upgrade ( run `decidim:upgrade` and `db:migrate` in dev app ) 
3. See changeset is correctly migrated and available
